psycop/common/model_training_v2/config/baseline_registry.py

```diff
@@ -0,0 +1,25 @@
+import catalogue
+from confection import registry
+
+
+class BaselineRegistry(registry):
+    baseline_loggers = catalogue.create("psycop", "baseline_loggers")
```

Comment:
The name-spacing is a bit funky (lots of `baseline_`) to avoid crashing with sequence models. Would love suggestions on how to make it simpler, perhaps another layer of nesting?

Reply:
just remove `baseline_`? Or does the catalogue.create("psycop", "loggers") lead to a error?
If that is the case it might be worth doing something like:
catalogue.create("psycop/baseline", "loggers")?